src/current/_includes/v24.2/backups/skip-localities-check-serverless.md

```diff
@@ -0,0 +1 @@
+Restoring a multi-region cluster into a single-region CockroachDB {{ site.data.products.serverless }} cluster is not supported. It is not possible to use the `skip-localities-check` option with `RESTORE`, because a {{ site.data.products.serverless }} cluster does not allow you to manually reconcile any restored regions that the cluster cannot meet. To restore a multi-region cluster, you must [create a new multi-region]({% link cockroachcloud/serverless-cluster-management.md %}#move-cluster-to-a-new-region) {{ site.data.products.serverless }} cluster with the desired regions that match the backup.
```

Comment:
This phrasing in the opening sentence confused me for a bit.  Maybe because it's after lunch, but I would find this easier to parse if it made it SUPER clear to me the average after lunch bad reader that this is ONLY about Serverless not anything else (correct?)

for example "it is not possible to use the skip-localities-check option with RESTORE" kind of sounds like it might not just be talking about Serverless, but I think it is talking about just Serverless?

something like the following (but better) would be easier IMO

> Restoring a multi-region cluster into a single-region cluster using `WITH skip_localities_check` is supported, _except_ on Serverless.  You cannot restore a multi-region Serverless cluster into a single-region Serverless cluster. To restore a multi-region Serverless cluster, you must create another new multi-region Serverless cluster.

I realize that says Serverless a lot (too much?) but IMO something like that would make it clearer to someone who isn't reading this carefully

Reply:
PS this is beyond our control ofc but this is one of those unfortunate situations where documenting an edge case on a less used version of our product is taking up the majority of textual space in this description of this option which has the result that the edge case seems "most important" because it's taking up the most room on the page

nothing to be done about it, just one of those #techwriterlife things

Reply:
I understand what you mean. I have edited somewhere in between. The creation of a new multi-region Serverless cluster is to accept the backup from the original MR cluster (does not necessarily have to be Serverless). It's just a user would need to create the *new* Serverless cluster with the correct regions, since they will not be able to skip locality checks. 

I have removed the line about reconciliation of the regions in Serverless. Instead, clearly pointing out that this is not supported, which is all that is necessary. I am also going to add to the `skip_localities_check` option description that the intention is for the user to then manually reconcile localities as needed (for other supported deployments).